DigitalLearningSolutions.Data.Tests/DataServices/UserDataServiceTests/DelegateUserDataServiceTests.cs

```diff
@@ -263,14 +263,30 @@ public void DeactivateDelegateUser_deactivates_delegate_user()
         {
             using var transaction = new TransactionScope();
 
-            // given
+            // Given
             userDataService.GetDelegateUserById(1)!.Active.Should().BeTrue();
 
-            // when
+            // When
             userDataService.DeactivateDelegateUser(1);
 
-            // then
+            // Then
             userDataService.GetDelegateUserById(1)!.Active.Should().BeFalse();
         }
+
+        [Test]
+        public void ReactivateDelegateUser_reactivates_delegate_user()

```

Comment:
Now that ReactivateDelegateUser has been renamed to ActivateDelegateUser, this test should probably be renamed in to be consistent with the new name.

Reply:
I have renamed all reactive to active